components/automate-ui/src/app/pages/automate-settings/automate-settings.component.html

```diff
@@ -139,10 +172,17 @@ <h2>Client Runs</h2>
             </div>
           
             <p>When no Chef Infra Client run data has been received from a node in
-              <input chefInput formControlName="threshold" [resetOrigin]="shouldResetValues"
-                type="number" min="0" (keydown)="preventNegatives($event.key)" />
+              <input
+                aria-label="Amount of time before labeling a nodes as missing"
```

Comment:
[nit]: sp - either
`aria-label="Amount of time before labeling nodes as missing"`
or
`aria-label="Amount of time before labeling a node as missing"`